src/app/cases/custom-case-export/custom-case-export.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component } from '@angular/core';
+import { CardAppearanceOptions } from '../../_constants/enums';
+
+@Component({
+  selector: 'app-custom-case-export',
+  templateUrl: './custom-case-export.component.html',
+  styleUrls: ['./custom-case-export.component.scss'],
+})
+export class CustomCaseExportComponent {
+  appearance = CardAppearanceOptions;
+  dataMyExports = [
+    {
+      title: 'Surveillance officer export for case data',
+      link: '/soefcd.pdf',
```

Comment:
translations needed with : this.translateService.instant(

Reply:
this is just dummy data, it will go away